azurerm/resource_arm_policy_assignment.go

```diff
@@ -90,6 +119,15 @@ func resourceArmPolicyAssignmentCreate(d *schema.ResourceData, meta interface{})
 		assignment.AssignmentProperties.Description = utils.String(v)
 	}
 
+	if v := d.Get("identity_type").(string); v != "" {
+		assignment.Identity = &policy.Identity{}
+		assignment.Identity.Type = policy.ResourceIdentityType(*(utils.String(v)))
+	}
+
+	if v := d.Get("location").(string); v != "" {
+		assignment.Location = utils.String(v)
```

Comment:
since it's possible for this location to have different values (e.g. `West Europe` vs `west europe` vs `westeurope`) we normalize this prior to sending it to azure (and on the return) - could we run this through the same normalization function `azureRMNormalizeLocation` here?


```suggestion
		assignment.Location = utils.String(azureRMNormalizeLocation(v))
```

Reply:
👍